vcd/resource_vcd_rde.go

```diff
@@ -0,0 +1,435 @@
+package vcd
+
+import (
```

Comment:
Could you run `goimports -w` on this and other newly created files?

Reply:
Done, changed all RDE fileset.